Sources/SwiftKueryPostgreSQL/PostgreSQLConnection.swift

```diff
@@ -586,3 +572,89 @@ public class PostgreSQLConnection: Connection {
         return nil
     }
 }
+
+class PostgreSQLColumnBuilder: ColumnCreator {
+    func buildColumn(for column: Column, using queryBuilder: QueryBuilder) -> String? {
+        guard let type = column.type else {
+            return nil
+        }
+
+        var result = column.name
+        let identifierQuoteCharacter = queryBuilder.substitutions[QueryBuilder.QuerySubstitutionNames.identifierQuoteCharacter.rawValue]
+        if !result.hasPrefix(identifierQuoteCharacter) {
+            result = identifierQuoteCharacter + result + identifierQuoteCharacter + " "
+        }
+
+        var typeString = type.create(queryBuilder: queryBuilder)
+        if let length = column.length {
+            typeString += "(\(length))"
+        }
+        if column.autoIncrement {
+            guard let autoIncrementType = getAutoIncrementType(for: typeString) else {
+                //Unrecognised type for autoIncrement column, return nil
+                return nil
```

Comment:
Same as above.

I'm not sure how important it'd be to alert a user to the reason why this has returned `nil`

Reply:
The nil is handled in the calling function and an error thrown that identifies the column as having invalid attributes.